docs(core): add crypto dashboard link to the project readme (#4099)
questdb/questdb: README.md

- Trades: live crypto market data with 30M+ rows per month
- Pos: geolocations of 250k unique ships over time

> For interactive real-time market data dashboards powered by QuestDB and Grafana, click [here](https://questdb.io/dashboards/crypto/)
> 

Reviewer: ```suggestion
> Checkout our [interactive real-time market data dashboards](https://questdb.io/dashboards/crypto/) powered by QuestDB and Grafana.

```
